build.gradle

```diff
@@ -1,180 +1,183 @@
 plugins {
-	id "java"
-	id "java-library"
-	id "checkstyle"
-	id "maven-publish"
-	id "com.diffplug.spotless" version "5.8.2"
-	id "me.modmuss50.remotesign" version "0.4.0"
+	id "com.diffplug.spotless" version "6.22.0" apply false
+	id "me.modmuss50.remotesign" version "0.4.0" apply false
 }
 
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+def ENV = System.getenv()
 
-group "net.fabricmc"
-archivesBaseName = "mapping-io"
+allprojects {
+	apply plugin: "java"
+	apply plugin: "java-library"
+	apply plugin: "maven-publish"
+	apply plugin: "checkstyle"
+	apply plugin: "com.diffplug.spotless"
+	apply plugin: "me.modmuss50.remotesign"
 
-def ENV = System.getenv()
+	group "net.fabricmc"
 
-tasks.withType(JavaCompile).configureEach {
-	it.options.encoding = "UTF-8"
+	tasks.withType(JavaCompile).configureEach {
+		it.options.encoding = "UTF-8"
 
-	if (JavaVersion.current().isJava9Compatible()) {
-		it.options.release = 8
+		if (JavaVersion.current().isJava9Compatible()) {
+			it.options.release = 8
+		}
 	}
-}
 
-checkstyle {
-	configFile = project.file("checkstyle.xml")
-	toolVersion = project.checkstyle_tool_version
-}
+	checkstyle {
+		configFile = rootProject.file("checkstyle.xml")
+		toolVersion = project.checkstyle_tool_version
+	}
 
-repositories {
-	maven {
-		name = 'Fabric'
-		url = 'https://maven.fabricmc.net/'
+	repositories {
+		mavenCentral()
 	}
-	mavenCentral()
-}
 
-dependencies {
-	implementation "org.ow2.asm:asm:${project.asm_version}"
-	compileOnly "org.jetbrains:annotations:${project.jetbrains_annotations_version}"
-	implementation "net.fabricmc:tiny-remapper:${tiny_remapper_version}"
+	dependencies {
+		implementation "org.ow2.asm:asm:${project.asm_version}"
+		compileOnly "org.jetbrains:annotations:${project.jetbrains_annotations_version}"
 
-	testImplementation "org.junit.jupiter:junit-jupiter-api:${junit_jupiter_version}"
-	testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:${junit_jupiter_version}"
-}
+		testImplementation "org.junit.jupiter:junit-jupiter:${junit_jupiter_version}"
+		testRuntimeOnly "org.junit.platform:junit-platform-launcher"
+	}
 
-spotless {
-	java {
-		licenseHeaderFile(rootProject.file("HEADER")).yearSeparator(", ")
+	spotless {
+		java {
+			licenseHeaderFile(rootProject.file("HEADER")).yearSeparator(", ")
+		}
 	}
-}
 
-java {
-	withSourcesJar()
-	withJavadocJar()
-}
+	java {
+		withSourcesJar()
+		withJavadocJar()
 
-test {
-	useJUnitPlatform()
-}
+		sourceCompatibility = 1.8
+		targetCompatibility = 1.8
+	}
 
-jar {
-	manifest {
-		attributes("Automatic-Module-Name": "net.fabricmc.mappingio")
+	test {
+		useJUnitPlatform()
 	}
-}
 
-javadoc {
-	failOnError = false
-}
+	javadoc {
+		failOnError = false
```

Comment:
Maybe add https://github.com/FabricMC/fabric-loader/blob/0ba1a7ee50f86a0bbfb2fc7b0a7b33db501211c5/build.gradle#L247-L248 too while you're at it? The log spam is quite annoying when building